coral/src/app/features/connectors/details/overview/ConnectorOverview.tsx

```diff
@@ -1,26 +1,21 @@
-import { Box, Label } from "@aivenio/aquarium";
+import { Box, PageHeader } from "@aivenio/aquarium";
 import MonacoEditor from "@monaco-editor/react";
 import { useConnectorDetails } from "src/app/features/connectors/details/ConnectorDetails";
 
 const ConnectorOverview = () => {
   const { connectorOverview } = useConnectorDetails();
-  // This is necessary to format the JSON renderd in MonacoEditor instead of rendering all in one line
-  const parsedConnectorInfo = JSON.stringify(
-    connectorOverview.connectorInfo,
-    null,
-    2
-  );
 
   return (
-    <Box marginTop={"l3"} marginBottom={"l2"}>
-      <Label>Connector</Label>
+    <>
+      <PageHeader title={"Connector configuration"} />
+
       <Box borderColor={"grey-20"} borderWidth={"1px"}>
         <MonacoEditor
           data-testid="topic-connector"
           height="300px"
           language="json"
           theme={"light"}
-          value={parsedConnectorInfo}
+          value={connectorOverview.connectorInfo.connectorConfig}
```

Comment:
We don't need to strigify this anymore because it is already a string?

Reply:
Yes, it is stored as a string on the `connectorInfo` object ^^